department-of-veterans-affairs/notification-api
# TESTING ONLY -  DO NOT MERGE UTILS-146 - update debug log level

TESTING ONLY DO NOT MERGE 

- Setup to test UTILS-146 locally -> Added log statement and updated dependency for `notification-utils` referencing its feature branch 146-update_debug_log_level.  Updated poetry.lock file.
- See https://github.com/department-of-veterans-affairs/notification-utils/pull/155
